src/local-history/local-history-manager.ts

```diff
@@ -33,7 +33,10 @@ export class LocalHistoryManager {
 
     constructor() {
         vscode.workspace.onDidSaveTextDocument((document: vscode.TextDocument) => {
-            this.saveEditorContext(document);
+            if (this.checkRevisionLimit(document) && this.localHistoryPreferencesService.fileLimit !== 0) {
```

Comment:
When will this be 0 if we have a minimum?

Reply:
Will fix it.